### PR TITLE
Added the Deal Dashboard Deal information card

### PR DESCRIPTION
**Pull request by IonelLupu**

### CODE REVIEW

File: src/dealDashboard/dealInfo/dealInfo.ts [hunk 1]
```diff
@@ -1,9 +1,39 @@
...
+      this.timeLeftToExecute = this.getTimeLeftToExecute();
+    }, 1000);
+    this.deal.loadDealSize();
+  }
+
+  getSocialMediaDetails(socialMediaName: string) {
+    return availableSocialMedias.find(socialMedia => socialMedia.name === socialMediaName);
+  }
+
+  getTimeLeftToExecute(): number | undefined {
```
Comment (dkent600) [inline src/dealDashboard/dealInfo/dealInfo.ts:29]: Can we move this into DealTokenSap?

File: src/services/DateService.ts [hunk 2]
```diff
@@ -300,8 +299,11 @@ export class DateService {
       // }
     }
 
-    if (!stop && !shouldStopOnLargest2(largestTwoCounter) && resolution <= TimespanResolution.seconds) {
-
+    if (!stop && !shouldStopOnLargest2(largestTwoCounter) && ((seconds ||
```
Comment (dkent600) [inline src/services/DateService.ts:302]: @hiaux0  Can you please review this?  I thought that code was already working.

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/services/DateService.ts (hunk 2 above)
Comment (hiaux0) [reply src/services/DateService.ts:302]: @dkent600 
As discussed, separate ticket 
https://github.com/PrimeDAO/prime-deals-dapp/issues/469

Comment (dkent600) [reply src/services/DateService.ts:302]: Can we revert this now that the bug is fixed elsewhere?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/dealDashboard/dealInfo/dealInfo.ts (hunk 1 above)
Comment (IonelLupu) [reply src/dealDashboard/dealInfo/dealInfo.ts]: Moved!

File: src/entities/DealTokenSwap.ts [hunk 3]
```diff
@@ -140,6 +140,13 @@ export class DealTokenSwap implements IDeal {
   //   return;
   // }
 
+  get timeLeftToExecute(): number | undefined {
+    if (!this.executedAt) {
```
Comment (dkent600) [inline src/entities/DealTokenSwap.ts:144]: use `this.isExecuted`

File: src/entities/DealTokenSwap.ts [hunk 4]
```diff
@@ -140,6 +140,13 @@ export class DealTokenSwap implements IDeal {
...
+      return;
+    }
+    return this.executedAt.getTime() + this.executionPeriod * 1000 - new Date().getTime();
```
Comment (dkent600) [inline src/entities/DealTokenSwap.ts:147]: Specify parentheses.

Use `Date.now()`; is cleaner, probably more efficient.

add @computedFrom

**Review by dkent600 — CHANGES_REQUESTED** [review]

**Review by dkent600 — APPROVED** [review]

**Pull request merged**